### DISCUSSION TITLE
Add sprite object size conditions

### DISCUSSION THREAD

**Discussion by jack-ii** [4ian/GDevelop#3530] [Archive] [answered]
- [x] Add an "Height" and "Width" condition in Sprite - Size.
- [x] Add an "Size" action for Sprite and Tiled-Sprite which can set Height and Width in the same action. Like e.g. in Position where can set X and Y at the same time.

**Comment by Silver-Streak** [comment, marked as answer]
Thanks for asking for these. These items went live with 5.0.129, so I'm going to close out this conversation.

**Reply by AlexandreSi** [reply]
After closing them, you can change their category for "Archive" to keep the other categories tidy.
